hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/freon/TestRandomKeyGenerator.java

```diff
@@ -156,28 +111,26 @@ void testRatisKey() {
   @Test
   void testKeyLargerThan2GB() {
     RandomKeyGenerator randomKeyGenerator =
-        new RandomKeyGenerator(cluster.getConf());
+        new RandomKeyGenerator(cluster().getConf());
     CommandLine cmd = new CommandLine(randomKeyGenerator);
     cmd.execute("--num-of-volumes", "1",
         "--num-of-buckets", "1",
         "--num-of-keys", "1",
         "--num-of-threads", "1",
         "--key-size", "2.01GB",
         "--factor", "THREE",
-        "--type", "RATIS",
-        "--validate-writes"
```

Comment:
why drop this?

Reply:
Use of the option `--validate-writes` has been moved to other test cases, which write less data:
- `testDefaultReplication`
- `testECKey`

The goal is to exercise this option, but we don't need to do that with the 2GB key.